src/doc/guide.md

```diff
@@ -136,68 +125,74 @@ project.
-    println!("HSV: {:?}", red.to_hsv::<f32>());
+    let re = Regex::new(r"^\d{4}-\d{2}-\d{2}$").unwrap();
+    println!("Did our date match? {}", re.is_match("2014-01-01"));
 }
 ```
 
-Let's tell Cargo to fetch this new dependency and update the `Cargo.lock`:
-
-<pre><code class="language-shell"><span class="gp">$</span> cargo update -p color
-<span style="font-weight: bold" class="s1">    Updating</span> git repository `https://github.com/bjz/color-rs.git`</code></pre>
+The next time we build, Cargo will fetch this new dependency, all of its
+dependencies, compile them all, and update the `Cargo.lock`:
 
-Compile it:
+<pre><code class="language-shell"><span class="gp">$</span> cargo build
+<span style="font-weight: bold" class="s1">    Updating</span> registry `https://github.com/rust-lang/crates.io-index`
+<span style="font-weight: bold" class="s1"> Downloading</span> memchr v0.1.5
+<span style="font-weight: bold" class="s1"> Downloading</span> libc v0.1.10
+<span style="font-weight: bold" class="s1"> Downloading</span> regex-synatx v0.2.1
+<span style="font-weight: bold" class="s1"> Downloading</span> memchr v0.1.5
+<span style="font-weight: bold" class="s1"> Downloading</span> aho-corasick v0.3.0
+<span style="font-weight: bold" class="s1"> Downloading</span> regex v0.1.41
+<span style="font-weight: bold" class="s1">   Compiling</span> memchr v0.1.5
+<span style="font-weight: bold" class="s1">   Compiling</span> libc v0.1.10
+<span style="font-weight: bold" class="s1">   Compiling</span> regex-synatx v0.2.1
+<span style="font-weight: bold" class="s1">   Compiling</span> memchr v0.1.5
+<span style="font-weight: bold" class="s1">   Compiling</span> aho-corasick v0.3.0
+<span style="font-weight: bold" class="s1">   Compiling</span> regex v0.1.41
+<span style="font-weight: bold" class="s1">   Compiling</span> foo v0.1.0 (file:///path/to/project/hello_world)
+
+Run it:
 
 <pre><code class="language-shell"><span class="gp">$</span> cargo run
-<span style="font-weight: bold" class="s1">   Compiling</span> color v0.1.0 (https://github.com/bjz/color-rs.git#bf739419)
-<span style="font-weight: bold" class="s1">   Compiling</span> hello_world v0.1.0 (file:///path/to/project/hello_world)
 <span style="font-weight: bold" class="s1">     Running</span> `target/hello_world`
-Converting RGB to HSV!
-HSV: HSV { h: 0, s: 1, v: 1 }</code></pre>
+Did our date match? true</code></pre>
 
-We just specified a `git` repository for our dependency, but our `Cargo.lock`
-contains the exact information about which revision we used:
-
-```toml
-[root]
-name = "hello_world"
-version = "0.1.0"
-dependencies = [
- "color 0.1.0 (git+https://github.com/bjz/color-rs.git#bf739419e2d31050615c1ba1a395b474269a4b98)",
-]
-
-[[package]]
-name = "color"
-version = "0.1.0"
-source = "git+https://github.com/bjz/color-rs.git#bf739419e2d31050615c1ba1a395b474269a4b98"
-
-```
+Our `Cargo.lock` contains the exact information about which revision of all of
+these dependencies we used:
```

Comment:
Ahh, so, I decided to remove the actual `toml` code, as we don't actually specify the format of it, and it's also much less illustrative than the `git` example. But, it shouldn't have a `:` at the end, but a `.`